components/Stats/widgets/WeatherStations.vue

```diff
@@ -101,7 +101,7 @@
   })
 
   const calcProgressBarPercentage = (percentage: number) => {
-    return percentage.toFixed(2) * 100
+    return (percentage * 100).toFixed(0)
```

Comment:
@tasos77 Could you provide a bit more context for this change? 
I'm assuming part of the change is also to ditch the decimal points entirely.

e.g.
```ts
const x = 1.001;

// Previous Case
x.toFixed(2) * 100
// => 100

// New change
(x * 100).toFixed(0)
// => '100'
```

So there is a change in the return type in the way this is invoked.
maybe something like `parseInt(percentage * 100, 10)` is more desirable as it will parse only the integer part and remain of type number.

p.s. This is probably nitpicking in the overall change so feel free to ignore it :P

Reply:
You have a point...
Well , the decimal trimming wasn't necessary from the beginning (in fact it is the result of a meaningless copy paste), bcs irrespective of the result the ui component (VProgressBar) trims the number by itself... 
But in this case for some reason the component is not able to trim the calced number so as an easy fix here we can trim the number right after the calculation...